bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -372,14 +383,14 @@ uint32_t PreProcessor::launchReqPreProcessing(uint16_t clientId, ReqId reqSeqNum
   return resultLen;
 }
 
+PreProcessingResult PreProcessor::getPreProcessingConsensusResult(uint16_t clientId) {
+  auto &clientEntry = ongoingRequests_[clientId];
+  lock_guard<mutex> lock(clientEntry->mutex);
+  return clientEntry->clientReqInfoPtr->getPreProcessingConsensusResult();
+}
```

Comment:
it's kind of confusing to have two methods called the same - getPreProcessingConsensusResult:
```
PreProcessingResult PreProcessor::getPreProcessingConsensusResult(uint16_t clientId) {
  auto &clientEntry = ongoingRequests_[clientId];
  lock_guard<mutex> lock(clientEntry->mutex);
  return clientEntry->clientReqInfoPtr->getPreProcessingConsensusResult();
}
```

Reply:
The first one is just a proxy to another.